### ISSUE TITLE
Please documentation and example app

### ISSUE THREAD

**Issue by aadarshadhakalg** [rikulo/entity#2]
I was trying to use this package in a hobby project, but I found this hard because I didn't know how to get started. There is no example app and documentation. Please add documentation.

**Comment by bradcypert** [comment]
Yeah, I was looking at ORM options for Dart (before considering rolling my own) and stumbled upon this. It looks actively updated, but the lack of docs is a dealbreaker for me.
